test/verify/check-shell-host-switching

```diff
@@ -526,5 +526,22 @@ class TestHostSwitching(testlib.MachineCase, HostSwitcherHelpers):
         self.assertNotIn(m2.execute("loginctl"), "admin")
 
 
+@testlib.todoPybridgeRHEL8()
+class TestNoHostSwitching(testlib.MachineCase, HostSwitcherHelpers):
```

Comment:
This doesn't provision a second machine, so 10.111.113.2 is not a thing.

Also, this is really expensive -- could this just go into `TestHostSwitching`, and restarting cockpit after the config change?

Reply:
> This doesn't provision a second machine, so 10.111.113.2 is not a thing.

That shouldn't matter, the code in question runs before there is any connection attempt.

> Also, this is really expensive -- could this just go into `TestHostSwitching`, and restarting cockpit after the config change?

Yes, or be a non-destructive test. Let's see...

Reply:
I have changed TestHostSwitching.testBasic to cover this.